### PR TITLE
CDK updates - overview page

### PR DESCRIPTION
**Pull request by kmurphypolygon**
Incorporating updates from @sharon 

For info, the second pic is referenced here: https://lh7-eu.googleusercontent.com/gusTl_jWu_eVU4RF32Y3LvQYG63nR9Ydi_qZQDNxTGyuhjlNsWvuRNMXJvr05bWZznFfedIu1smvHJsGP9nQgxbHipfpmrhQfN9dmvED20B4BSRI5fSP3MV3ztoYuqEAVIStTR-_Aynv7zVGLOPox9o

### CODE REVIEW

**Comment by Sharonbc01** [comment]
The changes are good by me thanks. I recommend you also have Gabriel Silva PM review these changes to ensure they are acceptable from a Product perspective before any merge. Thanks!

File: docs/cdk/overview.md [hunk 1]
```diff
@@ -1,15 +1,29 @@
-Polygon Chain Development Kit (CDK) is a modular, open source software toolkit for blockchain developers which supports the installation and configuration of a variety of chain architectures. Polygon CDK empowers developers to launch new L2 chains on Ethereum or, in the future, transition existing Layer 1 (L1) chains into L2s. 
+Polygon Chain Development Kit (CDK) is a modular, open source software toolkit for blockchain developers which supports the installation and configuration of a variety of chain architectures. Polygon CDK empowers developers to launch new L2 chains running Polygon zkEVM technology on Ethereum or, in the future, transition existing Layer 1 (L1) chains into ZK-EVM L2s. 
 
-With Polygon CDK, developers can select a chain architecture specific to their needs from a set of supported, open source components. Alternatively, developers can select custom components for specific requirements. The diagram below shows the two supported configuration options, rollup or validium. Rollups post transaction data from Polygon CDK directly onto Ethereum whereas validiums only post the transaction hash.
+!!! warning "ZK-EVM versus zkEVM"
```
Comment (hsutaiyu) [inline docs/cdk/overview.md]:
```suggestion
!!! warning "ZK-EVM versus zkEVM"
```

File: docs/cdk/overview.md [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
+    - Whenever we refer to **ZK-EVM** chains, this is meant in a general sense and includes any zero-knowledge rollup chain.
```
Comment (hsutaiyu) [inline docs/cdk/overview.md]:
```suggestion
    -  **ZK-EVM**: The capitalized and hyphenated usage of the term **refers to any zero-knowledge rollup chain in general**.
```

File: docs/cdk/overview.md [hunk 3]
```diff
@@ -1,15 +1,29 @@
...
+    - Whenever we are specifically referring to the Polygon zkEVM product, we use **zkEVM**.
```
Comment (hsutaiyu) [inline docs/cdk/overview.md]:
```suggestion
    - **zkEVM** refers to Polygon's zero-knowledge rollup and validium scaling solution.
```

File: docs/cdk/overview.md (hunk 2 above)
Comment (hsutaiyu) [reply docs/cdk/overview.md]: Suggest avoiding the use of "we"

File: docs/cdk/overview.md (hunk 3 above)
Comment (hsutaiyu) [reply docs/cdk/overview.md]: Suggest avoiding the use of "we"

File: docs/cdk/overview.md [hunk 4]
```diff
@@ -1,15 +1,29 @@
...
+
+## CDK options
+
+With Polygon CDK, developers can select a chain architecture specific to their needs from a set of supported, open source components. Alternatively, developers can select custom components for specific requirements. The diagram below shows the two supported configuration options, zkEVM rollup or validium. zkEVM rollups post transaction data from Polygon CDK directly onto Ethereum whereas validiums only post the transaction hash.
```
Comment (hsutaiyu) [inline docs/cdk/overview.md]:
```suggestion
With Polygon CDK, developers can select a chain architecture specific to their needs from a set of supported, open source components. Alternatively, developers can select custom components for specific requirements. The diagram below shows the two supported configuration options, zkEVM rollup or validium. zkEVM rollups post transaction data from Polygon CDK directly onto Ethereum, whereas validiums only post the transaction hashes, while storing transaction data off-chain.
```

File: docs/cdk/overview.md [hunk 5]
```diff
@@ -1,15 +1,29 @@
...
 
-A chain tailored for a specific application might leverage the zkEVM execution environment, adopt the Validium mode, and implement a centralized sequencer. 
+A chain tailored for a specific application might leverage the zkEVM execution environment, adopt the validium mode, and implement a centralized sequencer. 
+
+!!! note "Example use case"
+     A CDK zkEVM rollup L2 chain dedicated to running a game has lower security requirements than a high-value DeFi application. This could also use a validium mode for off-chain data availability to further reduce transaction costs.
 
-Regardless of these individualized choices, after the completion of the in-development interop layer, every chain will have the option to be interoperable with other Polygon network chains.
+!!! info "Coming soon"
+     Upon the completion of the aggregation layer every chain will have the option to be interoperable with other Polygon network chains. For now the aggregation layer in the CDK kit allows for independent chains (1 to 1) to settle independently.
```
Comment (hsutaiyu) [inline docs/cdk/overview.md]:
```suggestion
     Upon the completion of the aggregation layer, every chain will have the option to be interoperable with other Polygon network chains. For now, the aggregation layer in the CDK kit allows for independent chains (1-to-1) to settle independently.
```

Comment (hsutaiyu) [reply docs/cdk/overview.md]: Maybe slightly rephrase for clarity?

File: docs/img/cdk/overview-1.png
Comment (hsutaiyu) [inline docs/img/cdk/overview-1.png:1]: Do the chains individually connect to the Prover, or does it only interact directly with AggLayer to generate aggregated proofs? I guess both are possible?

**Review by hsutaiyu — APPROVED** [review]
Approved with important suggestions

File: docs/cdk/overview.md (hunk 4 above)
Comment (hsutaiyu) [reply docs/cdk/overview.md]: Might wanna confirm and double-check the accuracy of these couple of statements

File: docs/cdk/overview.md [hunk 6]
```diff
@@ -1,15 +1,29 @@
...
+Polygon Chain Development Kit (CDK) is a modular, open source software toolkit for blockchain developers which supports the installation and configuration of a variety of chain architectures. Polygon CDK empowers developers to launch new L2 chains running Polygon zkEVM technology on Ethereum or, in the future, transition existing Layer 1 (L1) chains into ZK-EVM L2s. 
 
-With Polygon CDK, developers can select a chain architecture specific to their needs from a set of supported, open source components. Alternatively, developers can select custom components for specific requirements. The diagram below shows the two supported configuration options, rollup or validium. Rollups post transaction data from Polygon CDK directly onto Ethereum whereas validiums only post the transaction hash.
+!!! warning "ZK-EVM versus zkEVM - term usage"
+    -  **ZK-EVM**: The capitalized and hyphenated usage of the term **refers to any zero-knowledge rollup chain in general**.
```
Comment (kmurphypolygon) [inline docs/cdk/overview.md]:
```suggestion
    -  **ZK-EVM**: The capitalized and hyphenated usage of the term refers to any zero-knowledge rollup chain in general.
```

**Pull request merged**